sklearn/metrics/ranking.py

```diff
@@ -439,9 +439,11 @@ class or confidence values.
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
-    drop_intermediate : boolean, optional (default=True)
+    drop_intermediate : boolean, optional (default=False)
         Whether to drop some suboptimal thresholds which would not appear
-        on a plotted ROC curve.
+        on a plotted ROC curve. This is useful in order to create lighter
+        ROC curves, but might cause problems for example while creating
```

Comment:
Indeed memory is not the only problem: many redundant point will slow down SVG rendering of ROC curves in a dashboard for instance.

So +1 for lighter.